### PR TITLE
Netty: Use optimal native transport options where available

### PR DESCRIPTION
**Pull request by Hc747**
Addresses: https://github.com/ktorio/ktor/issues/1122

### CODE REVIEW

**Comment by Hc747** [comment]
Unfortunately, this PR is not ready for implementation as I could not figure out how to include the required dependencies! 😢

**Comment by Hc747** [comment]
Should be ready for review: https://github.com/ktorio/ktor/pull/1124 @cy6erGn0m

File: ktor-server/ktor-server-netty/jvm/src/io/ktor/server/netty/NettyApplicationEngine.kt [hunk 1]
```diff
@@ -8,13 +8,22 @@ import io.ktor.application.*
 import io.ktor.server.engine.*
 import io.ktor.util.*
 import io.ktor.util.pipeline.*
-import io.netty.bootstrap.*
-import io.netty.channel.*
-import io.netty.channel.nio.*
-import io.netty.channel.socket.nio.*
+import io.netty.bootstrap.ServerBootstrap
```
Comment (cy6erGn0m) [inline ktor-server/ktor-server-netty/jvm/src/io/ktor/server/netty/NettyApplicationEngine.kt]: Use star imports instead

**Comment by Hc747** [comment]
@cy6erGn0m Is there anything further I can do to assist with this PR?

**Comment by cy6erGn0m** [comment]
I believe it will be integrated into the master soon. Thanks for your contribution!

**Comment by cy6erGn0m** [comment]
Merged to master. Will be available in 1.2.2

**Pull request closed without merging**